phpstan.dist.neon

```diff
@@ -5,22 +5,56 @@ includes:
     - dev-tools/phpstan/baseline.php
 
 parameters:
-    level: 6
+    level: 7
     paths:
         - src
         - tests
     excludePaths:
         - tests/Fixtures
+    polluteScopeWithLoopInitialAssignments: true # Do not enforce assignments outside of the loops
```

Comment:
OKish 😅

Reply:
It's basically something we rely on in our code, as we often have loops where we iterate `$index`, change it within loop, and then operate on its value outside of the loop. We would need to refactor a lot of stuff to threat loops as separate scope.